### ISSUE TITLE
Implement the user manager

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#42]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

The user manager class.

### Why

To allow the app's client code to handle high-level user operations.

### Issue expands upon

This issue expands upon another issue(s): #41

### Blockers

This issue is blocked by the following issue(s): #`issue number`
